nischalshrestha/CogulatorPlus
# Feature/button: To log min/max/average times

- Created a new layer that contains the button, averageTimeButton which can be referenced as a symbol in Main now; this symbol will also now appear in the library under average_time folder
- Created custom AverageTimeButton for custom behavior if needed
- Linked button to a handler, onAverageClick() that runs the averageModelTimes() function
- Added boolean variable to guard against possible spamming of the button or shortcut

To test:

Run the app and on the output you should see "AverageTimeButton" created when app starts up. Click the button and you will see the output of the function.
